doc/api/next_api_changes/deprecations/18226-ES.rst

```diff
@@ -0,0 +1,12 @@
+Widget class internals
+~~~~~~~~~~~~~~~~~~~~~~
+
+Several `.widgets.Widget` class internals have been privatized and deprecated:
```

Comment:
Do we think that there was anyone directly adding things to the `observers` dictionary?

Reply:
Then they _should_ have updated `cnt` so that followup additions would have the right number. I would expect them to get an error from that, and hopefully figure it out.